index.html

```diff
@@ -165,8 +165,8 @@
                 <p class="listing__price">$<%- parseInt(listItem.price.Price).formatMoney(0) %></p>
 
                 <div class="listing__details-lower">
-                  <p class="listing__location listing__location--short"><%- listItem.propertyType[0] %> • <%- listItem.address.AddrNeighborhood %></p>
-                  <p class="listing__location listing__location--full"><%- listItem.address.AddrDisplay %> • <%- listItem.address.AddrNeighborhood + ', ' + listItem.address.AddrStateAbbr %></p>
+                  <p class="listing__location listing__location--short"><%- listItem.propertyType[0] %> • <%- listItem.address.AddrCity %></p>
```

Comment:
Just might want to add some redundancy in case these aren't defined, which I think I've seen state not be defined more than once.